### ISSUE TITLE
`wrapSingleLineComments` applies to multiline comments as well

### ISSUE THREAD

**Issue by awein** [nicklockwood/SwiftFormat#1270]
The new `wrapSingleLineComments` rule introduced with  #1151 is transforming this comment
```
    /// See [Visual Format Language](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/VisualFormatLanguage.html)
    /// in Apple's documentation.
```

into

```
    /// See [Visual Format
    // Language](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/VisualFormatLanguage.html)
    /// in Apple's documentation.
```

**Comment by choulepoka** [comment]
And here is another side-effect:

```
loadPreviousSubject
            .removeDuplicates() // Little trick, when prefetching, the requested row index is sent, and if we receive 2-times the same index, we know we can discard the 2nd invocation, because we just loaded it
```

Became:
```
loadPreviousSubject
            .removeDuplicates() // Little trick, when prefetching, the requested row index is sent, and if we receive 2-times the same index, we know we can discard the 2nd invocation, because we just
            .removeDuplicates() // loaded it
```

For now, I will disable the rule itself, because it's not ready

**Comment by pronebird** [comment]
The other issue with this that it ignores triple slash `///` and breaks with `//`

<img width="962" alt="Screenshot 2022-09-29 at 13 26 05" src="https://user-images.githubusercontent.com/704044/193019495-716d6983-d2e5-4725-97b6-9ec73de7779b.png">

**Comment by nicklockwood** [comment]
Thanks all, I'll get these issues fixed asap

**Comment by nicklockwood** [comment]
@pronebird @choulepoka I've fixed your issues in 0.50.1

@awein I don't have a solution for this case yet. The best option is probably to disable the `wrapSingleLineCommentsRule` for now.

**Comment by nicklockwood** [comment]
@awein @orchetect fixed in 0.50.3
